### PR TITLE
Add manually curated 1.11.1/2/3 changelog

### PR DESCRIPTION
**Pull request by jgehrcke**
The goal is to introduce a changelog. These PRs are all related:

* **Curated 1.10 changelog**, no PR template modification:
    DC/OS Enterprise: https://github.com/mesosphere/dcos-enterprise/pull/2135 (update 180419: has been merged)
    DC/OS: https://github.com/dcos/dcos/pull/2364 (update 180419: has been merged)

* **Empty 1.12 changelog**, PR template modification to make changelog entries _obligatory_:
    DC/OS Enterprise: https://github.com/mesosphere/dcos-enterprise/pull/2134 (update 180419: merged)
    DC/OS: https://github.com/dcos/dcos/pull/2366 (update 180419: merged)

* **Empty 1.11 changelog**, no PR template mod:
  DC/OS Enterprise: https://github.com/mesosphere/dcos-enterprise/commit/d7147e8a19cc3e2a92d051c84168082a1575e532 (update 180419: https://github.com/mesosphere/dcos-enterprise/pull/2628)
  DC/OS: https://github.com/dcos/dcos/pull/2365 (update 180419: added manually curated 1.11.1 changelog)

### CODE REVIEW

**Comment by jgehrcke** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
I did this by going through commits and JIRA.

In our public release notes (HTML) for 1.11.1 we missed to mention the following items entirely:

* DC/OS log: consolidated handling of journald log file rotation. (DCOS_OSS-2132)

* Improved error reporting when sanity checks fail after an upgrade. (DCOS_OSS-2028)

* Consolidated Marathon's configuration for GPU resources. (MARATHON-8090)

* Moved Prometheus producer to port 61091. (DCOS-21545)

* Cosmos: Fixed a crash upon uninstalling Marathon apps that don't define `env`. (DCOS-21374)

* Fixed a bug where the Admin Router would not pick up Mesos leader changes (leading to unexpected 404 HTTP responses when using the service endpoint). (DCOS-21451)

**Comment by mesosphere-mergebot2** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by jgehrcke** [comment]
I have now added a curated 1.11.2 changelog, too.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
This now is the best comprehensive and curatend 1.11.1/2/3 changelog that I know of, and as of writing it it is up to date with the head of the `1.11` branches of both DC/OS repositories.

Coincidentally there is not 1.11 merge train in flight.

Today is the perfect point in time to go ahead with this pair of PRs. We should merge them today and then _actually require_ additions to the `CHANGES.md` file with every PR against the `1.11` branch.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot bump-ee

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+### Security updates
+
+
+## DC/OS 1.11.2
+
+
+### Notable changes
+
+* Updated to [Mesos 1.5.1-dev](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
```
Comment (gpaul) [inline CHANGES.md]: Upgrading to a `-dev` release seems pretty weird, but I guess such is life.

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+* Updated to [Marathon 1.6.392](https://github.com/dcos/dcos/pull/2678).
+
+* Updated to [Metronome 0.4.2](https://github.com/dcos/metronome/releases/tag/v0.4.2).
```
Comment (gpaul) [inline CHANGES.md]: For example, take this beautiful URL

File: CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+### Fixed and improved
+
+* DC/OS Metrics: metric names are now sanitized for better compatibility with Prometheus. (DCOS_OSS-2360)
+
+* Reverted the Marathon configuration change for GPU resources which was introduced with the 1.11.1 release. (MARATHON-8090)
+
+* The IP detect script and fault domain detect script may now be changed with a config upgrade. (DCOS-21611)
+
+* Upgraded Erlang/OTP runtime to address a race condition in TLS connection establishment. (DCOS_OSS-2378)
```
Comment (gpaul) [inline CHANGES.md]: 80

File: CHANGES.md (hunk 1 above)
Comment (gpaul) [reply CHANGES.md]: This appears to be an artifact of our having a different release cadence to Mesos. That makes total sense.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot ship-it-label-status-check

**Comment by jgehrcke** [comment]
@mesosphere-mergebot merge-it

File: CHANGES.md (hunk 1 above)
Comment (orsenthil) [reply CHANGES.md]: We could actually request mesos team (@rukletsov / @karya0 / @greggomann ) to consider providing a tagged URI like (Metronome).

Comment (rukletsov) [reply CHANGES.md]: I think in the future we should reference commits from the apache/mesos repo and if possible released versions.

**Pull request merged**